packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.html

```diff
@@ -4,21 +4,28 @@
       <slot></slot>
       <div
         class="tab-bar_more-button slds-tabs_default__item slds-tabs_default__overflow-button slds-is-absolute slds-p-right_none slds-p-left_none"
-        title={labels.moreTabs} role="presentation">
+        title={labels.moreTabs}
+        role="presentation"
+      >
         <div class={dropdownClasses}>
-          <button class="slds-button slds-var-p-horizontal_small" aria-haspopup="true"
-            onclick={toggleDropdown}>{moreButtonLabel}
-            <lightning-icon class={moreButtonIconClasses} icon-name={arrowIconName} size="xx-small" aria-hidden="true">
-            </lightning-icon>
+          <button class="slds-button slds-var-p-horizontal_small" aria-haspopup="true" onclick={toggleDropdown}>
+            {moreButtonLabel}
+            <lightning-icon class={moreButtonIconClasses} icon-name={arrowIconName} size="xx-small" aria-hidden="true"> </lightning-icon>
           </button>
           <div class="slds-dropdown slds-dropdown_right">
-            <ul class="slds-dropdown__list slds-dropdown_length-with-icon-10" role="menu">
+            <ul class="slds-dropdown__list slds-dropdown_length-with-icon-10" role="none">
               <template for:each={tabsInDropdown} for:item="tab">
-                <li class="slds-dropdown__item" role="presentation" key={tab.id}>
-                  <a role="menuitem" tabindex={optionTabIndex} onclick={handleDropdownTabSelect} data-value={tab.value}
-                    data-label={tab.label}>
-                    <span class="slds-truncate" title={tab.label}>{tab.label}</span>
-                  </a>
+                <li class="slds-dropdown__item" role="none" key={tab.id}>
+                  <button
+                    class="slds-button slds-tabs_default__item"
+                    tabindex={optionTabIndex}
+                    onclick={handleDropdownTabSelect}
+                    data-value={tab.value}
+                    data-label={tab.label}
+                    aria-pressed="false"
```

Comment:
Is it always false? It's because when you press it, it'll come out of the dropdown?